spec/support/schemas/claims_api/v2/errors/default_with_source.json

```diff
@@ -0,0 +1,28 @@
+{
+  "required": ["errors"],
+  "properties": {
+    "errors": {
+      "type": "array",
+      "items": {
+        "additionalProperties": false,
+        "required": ["status", "detail", "source"],
+        "properties": {
+          "status": {
+            "type": "string",
+            "description": "HTTP error code"
+          },
+          "detail": {
+            "type": "string",
+            "description": "HTTP error detail"
+          },
+          "source": {
+            "type": "string"
+          },
+          "title": {
+            "type": "string"
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
I believe the addition of this file is needed since the schema structure is changing for this (by adding in `title`) which would change the v1 output for this if we updated the shared file.